files/goquorum/config/goquorum/data/qbft-enhanced-genesis.json

```diff
@@ -8,44 +8,52 @@
     "eip158Block": 0,
     "byzantiumBlock": 0,
     "constantinopleBlock": 0,
-    "istanbul": {
-      "epoch": 30000,
-      "policy": 0,
-      "testQBFTBlock": 0,
-      "ceil2Nby3Block": 0
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "qbft": {
+      "epochLength": 30000,
+      "blockPeriodSeconds": 5
     },
-    "txnSizeLimit": 64,
-    "maxCodeSizeConfig" : [
+    "transitions": [
       {
-        "block" : 0,
-        "size" : 64
+        "block": 0,
+        "transactionSizeLimit": 64,
+        "contractSizeLimit": 64,
+        "privacyEnhancementsEnabled": true
+      },
+      {
+        "block": 10,
+        "blockPeriodSeconds": 1
+      },
+      {
+        "block": 20,
+        "emptyBlockPeriodSeconds": 60
       }
     ],
-    "isQuorum": true,
-    "privacyEnhancementsBlock": 0
```

Comment:
we should put this back `privacyEnhancementsBlock` as this is an `enhanced-genesis`

Reply:
it's in the transition at block 0 😄